docs/java/index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Java ♡ Ethereum
+lang: en-US
+sidebar: auto
+sidebarDepth: 0
+
+---
+# Java ♡ Ethereum
+
+<div class="featured">Learn how to develop for Ethereum using Java-based projects and tooling</div><br>
+
+Use Ethereum to create decentralized applications (or “dapps”) that utilize the benefits of cryptocurrency and blockchain technology. These dapps can be trustworthy, meaning that once they are "uploaded" to Ethereum, they will always run as programmed. They can control digital assets in order to create new kinds of financial applications. They can be decentralized, meaning that no single entity or person controls them and are nearly impossible to censor.
```

Comment:
Minor suggestion - "deployed" vs. "uploaded"? "Deploy" feels like nomenclature developers will be more familiar with.

Reply:
Fixed!